hwdb/20-OUI.hwdb

```diff
@@ -591,7 +591,7 @@ OUI:0000C2*
  ID_OUI_FROM_DATABASE=INFORMATION PRESENTATION TECH.
 
 OUI:0000C3*
- ID_OUI_FROM_DATABASE=HARRIS CORP COMPUTER SYS DIV
+ ID_OUI_FROM_DATABASE=Harris Corporation
```

Comment:
So all the sorting is paying off. It seems like again, there are just corrections and additions, no major removals.